.github/workflows/create_release.yml

```diff
@@ -0,0 +1,58 @@
+name: Create GitHub Release for Extension Package
+
+on:
+    workflow_dispatch:
+
+jobs:
+    release:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+
+            - name: Install Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: '16.x'
+
+            - name: Install dependencies
+              run: npm install
+
+            - name: Build VSIX Package
+              run: npx vsce package
+
+            - name: Get package version
```

Comment:
Realizing after doing more of this script, that I could just inline this specific step to the "Create release" step below. I'll make that update, as I think that will be cleaner.